Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM maven:latest
+
+WORKDIR /code
+
+COPY . /code
+
+ARG DB_HOST
+ARG DB_PASSWORD
+ARG DB_USERNAME
+ARG DB_NAME
+
+ENV DB_HOST ${DB_HOST}
+ENV DB_PASSWORD ${DB_PASSWORD}
+ENV DB_USERNAME ${DB_USERNAME}
+ENV DB_NAME ${DB_NAME}
+
+RUN mvn clean install -DskipTests=true
+
+EXPOSE 8080
+EXPOSE 8081
```

Comment:
Is this needed? Are you going to be running the application via docker locally? As this won't get exposed on AWS

Reply:
Just thought for testing purpose. Might be handy down the line? I'm happy to drop... @glenn-hamilton

Reply:
I think in that case it can be removed from the `Dockerfile`. As if you need the `healthcheck` endpoint then the application can be run outside of `Docker`